<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Settings {
    background: #1a1412;
}

QLabel {
    background: transparent;
    font-family: &quot;Press Start 2P&quot;, &quot;Courier New&quot;, monospace;
}

QGroupBox {
    background: transparent;
    border: 2px solid #2a1d19;
    border-radius: 0px;
    margin-top: 16px;
    padding-top: 12px;
    padding-left: 12px;
    font-size: 10px;
    letter-spacing: 0px;
    text-transform: uppercase;
    color: #8d7b68;
    font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 8px;
    top: -8px;
    padding: 0 4px 0 4px;
    background: #1a1412;
}

QScrollArea {
    background: transparent;
    border: none;
}

QScrollBar:vertical {
    background: #14100f;
    width: 12px;
    border: 2px solid #2a1d19;
    border-radius: 0px;
}

QScrollBar::handle:vertical {
    background: #3d2b25;
    border: 1px solid #5a4339;
    border-radius: 0px;
    min-height: 30px;
}

QScrollBar::handle:vertical:hover {
    background: #5a4339;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}

QTextEdit {
    background: #14100f;
    border: 1px solid #2a1d19;
    color: #e8d5c4;
    padding: 8px;
    font-family: &quot;Courier New&quot;, monospace;
    font-size: 11px;
}

QTextEdit:focus {
    border: 1px solid #c4a77d;
}

QPlainTextEdit {
    background: #14100f;
    border: 1px solid #2a1d19;
    color: #e8d5c4;
    padding: 8px;
    font-family: &quot;Courier New&quot;, monospace;
    font-size: 11px;
}

QPlainTextEdit:focus {
    border: 1px solid #c4a77d;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="contentWidget">
     <property name="styleSheet">
      <string notr="true">QWidget#contentWidget {
    background: #1a1412;
}</string>
     </property>
     <layout class="QVBoxLayout" name="contentLayout">
      <property name="spacing">
       <number>24</number>
      </property>
      <property name="leftMargin">
       <number>48</number>
      </property>
      <property name="topMargin">
       <number>40</number>
      </property>
      <property name="rightMargin">
       <number>48</number>
      </property>
      <property name="bottomMargin">
       <number>40</number>
      </property>
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string>SETTINGS</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #c4a77d;
    font-size: 18px;
    font-weight: bold;
    letter-spacing: 1px;
    background: transparent;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QScrollArea {
    border: none;
}</string>
        </property>
        <widget class="QWidget" name="scrollContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>820</width>
           <height>520</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="scrollLayout">
          <property name="spacing">
           <number>24</number>
          </property>
          <item>
           <widget class="QGroupBox" name="privacyGroup">
            <property name="title">
             <string>PRIVACY POLICY</string>
            </property>
            <layout class="QVBoxLayout" name="privacyLayout">
             <property name="spacing">
              <number>12</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item>
              <widget class="QLabel" name="privacyLabel">
               <property name="text">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p style=&quot;color: #e8d5c4; font-size: 11px; line-height: 1.6;&quot;&gt;&lt;b&gt;Privacy Policy:&lt;/b&gt; We respect your privacy. This application does NOT collect any user data, except when you voluntarily submit feedback. All audio processing is performed locally on your device.&lt;/p&gt;&lt;p style=&quot;color: #e8d5c4; font-size: 11px; line-height: 1.6;&quot;&gt;&lt;b&gt;Author:&lt;/b&gt; @3uyuan1ee | &lt;b&gt;License:&lt;/b&gt; MIT&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="openExternalLinks">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="privacyButtonsLayout">
               <item>
                <widget class="QPushButton" name="btnViewPrivacy">
                 <property name="minimumSize">
                  <size>
                   <width>160</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>VIEW PRIVACY POLICY</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background: #5a4339;
    color: #e8d5c4;
    border: 2px solid #8d7b68;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    padding: 6px 12px;
}

QPushButton:hover {
    background: #8d7b68;
    color: #c4a77d;
    border-color: #c4a77d;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
    border-color: #c4a77d;
}</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="privacyButtonsSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="configGroup">
            <property name="title">
             <string>CONFIGURATION</string>
            </property>
            <layout class="QVBoxLayout" name="configLayout">
             <property name="spacing">
              <number>12</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item>
              <widget class="QPlainTextEdit" name="configTextEdit">
               <property name="minimumHeight">
                <number>200</number>
               </property>
               <property name="placeholderText">
                <string>Configuration will be loaded here...</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="configButtonsLayout">
               <property name="spacing">
                <number>12</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnReloadConfig">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>28</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>RELOAD</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background: #1a1412;
    color: #c4a77d;
    border: 2px solid #3d2b25;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    padding: 4px 12px;
}

QPushButton:hover {
    background: #3d2b25;
    color: #e8d5c4;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
}</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnSaveConfig">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>28</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>SAVE</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background: #3d2b25;
    color: #e8d5c4;
    border: 2px solid #5a4339;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    padding: 4px 12px;
}

QPushButton:hover {
    background: #5a4339;
    color: #c4a77d;
    border-color: #c4a77d;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
    border-color: #c4a77d;
}</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="configButtonsSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="aboutGroup">
            <property name="title">
             <string>ABOUT &amp; UPDATES</string>
            </property>
            <layout class="QVBoxLayout" name="aboutLayout">
             <property name="spacing">
              <number>12</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="versionInfoLayout">
               <item>
                <widget class="QLabel" name="versionLabel">
                 <property name="text">
                  <string>Version: 1.0.0</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: #e8d5c4;
    font-size: 12px;
    background: transparent;</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="versionInfoSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="btnCheckUpdate">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>28</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>CHECK UPDATES</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background: #1a1412;
    color: #c4a77d;
    border: 2px solid #3d2b25;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    padding: 4px 12px;
}

QPushButton:hover {
    background: #3d2b25;
    color: #e8d5c4;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
}</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnOpenGitHub">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>28</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>GITHUB</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background: #1a1412;
    color: #c4a77d;
    border: 2px solid #3d2b25;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    padding: 4px 12px;
}

QPushButton:hover {
    background: #3d2b25;
    color: #e8d5c4;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
}</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="updateStatusLabel">
               <property name="text">
                <string>Click &quot;Check Updates&quot; to see if a new version is available.</string>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #6b5c52;
    font-size: 10px;
    background: transparent;</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="feedbackGroup">
            <property name="title">
             <string>FEEDBACK</string>
            </property>
            <layout class="QVBoxLayout" name="feedbackLayout">
             <property name="spacing">
              <number>12</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item>
              <widget class="QLabel" name="feedbackLabel">
               <property name="text">
                <string>Found a bug? Have a suggestion? Send feedback to help improve this application.</string>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #e8d5c4;
    font-size: 11px;
    background: transparent;</string>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="feedbackButtonsLayout">
               <property name="spacing">
                <number>12</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnSendFeedback">
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>SEND FEEDBACK</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background: #3d2b25;
    color: #e8d5c4;
    border: 2px solid #5a4339;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    padding: 6px 12px;
}

QPushButton:hover {
    background: #5a4339;
    color: #c4a77d;
    border-color: #c4a77d;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
    border-color: #c4a77d;
}</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="feedbackButtonsSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="chkIncludeLogs">
               <property name="text">
                <string>Include logs and usage statistics in feedback</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QCheckBox {
    color: #e8d5c4;
    font-size: 11px;
    background: transparent;
}

QCheckBox::indicator {
    width: 16px;
    height: 16px;
    border: 2px solid #5a4339;
    background: #1a1412;
    border-radius: 0px;
}

QCheckBox::indicator:checked {
    background: #c4a77d;
    border-color: #c4a77d;
}</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="licenseGroup">
            <property name="title">
             <string>LICENSES</string>
            </property>
            <layout class="QVBoxLayout" name="licenseLayout">
             <property name="spacing">
              <number>12</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>20</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>12</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="licenseButtonsLayout">
               <property name="spacing">
                <number>12</number>
               </property>
               <item>
                <widget class="QPushButton" name="btnViewLicense">
                 <property name="minimumSize">
                  <size>
                   <width>140</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>VIEW LICENSE</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background: #5a4339;
    color: #e8d5c4;
    border: 2px solid #8d7b68;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    padding: 6px 12px;
}

QPushButton:hover {
    background: #8d7b68;
    color: #c4a77d;
    border-color: #c4a77d;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
    border-color: #c4a77d;
}</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnViewThirdPartyLicense">
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>VIEW THIRD-PARTY LICENSES</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
    background: #5a4339;
    color: #e8d5c4;
    border: 2px solid #8d7b68;
    border-radius: 0px;
    font-size: 11px;
    font-weight: bold;
    padding: 6px 12px;
}

QPushButton:hover {
    background: #8d7b68;
    color: #c4a77d;
    border-color: #c4a77d;
}

QPushButton:pressed {
    background: #c4a77d;
    color: #1a1412;
    border-color: #c4a77d;
}</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="licenseButtonsSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="bottomSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
